Makefile

```diff
@@ -159,6 +159,9 @@ kind-run:
 		${KIND_CLUSTER}-control-plane \
 		bash -c "cd ${GOPATH}/src/istio.io/installer; make ${TARGET}"
 
+# Runs helm lint on the various charts
+lint:
```

Comment:
this target already exists, not sure how make will handle this

Reply:
I rebased - must have missed htis.  Will address.